// ngx-material-file-input/esm2020/lib/material-file-input.module.mjs
import { NgModule } from '@angular/core';
import { FocusMonitor } from '@angular/cdk/a11y';
import { FileInputComponent } from './file-input/file-input.component';
import { ByteFormatPipe } from './pipe/byte-format.pipe';
import * as i0 from "@angular/core";
export class MaterialFileInputModule {
}
MaterialFileInputModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.0.1", ngImport: i0, type: MaterialFileInputModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
MaterialFileInputModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.0.1", ngImport: i0, type: MaterialFileInputModule, declarations: [FileInputComponent, ByteFormatPipe], exports: [FileInputComponent, ByteFormatPipe] });
MaterialFileInputModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.0.1", ngImport: i0, type: MaterialFileInputModule, providers: [FocusMonitor] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.0.1", ngImport: i0, type: MaterialFileInputModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FileInputComponent, ByteFormatPipe],
                    providers: [FocusMonitor],
                    exports: [FileInputComponent, ByteFormatPipe]
                }]
        }] });
